GUI/Types/Renderer/ModelSceneNode.cs

```diff
@@ -116,6 +131,19 @@ public override void Update(Scene.UpdateContext context)
                 }
 
                 LocalBoundingBox = newBoundingBox;
+
+                var datas = AnimationController.GetDatas();
```

Comment:
this doesn't need to be in try/finally as it doesn't access matrices.

that being said, calling render from update isn't a very good idea.

Reply:
What would be a good place for rendering them?

Reply:
Realistically it should be in Render(), but batch renderer doesn't call that for models as the comment implies, maybe that can be changed.